### PR TITLE
Copy constructor chapter

### PR DESCRIPTION
**Pull request by zeddie888**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,17 @@
 # cpp-boot-camp
 
-These pages are intended to cover essential C and C++ topics, using a mix of notes and code samples
+These chapters are intended to cover essential C and C++ topics, using a mix of notes and code samples
 
-Content is derived from multiple sources, mainly from the University of Washington's [CSE 333](https://courses.cs.washington.edu/courses/cse333/) (Systems Programming) course
+Content is derived from multiple sources, including the University of Washington's [CSE 333](https://courses.cs.washington.edu/courses/cse333/) (Systems Programming) course
 
-Also (hopefully) a nice read for new [Husky Satellite Lab](https://huskysat.org/) members!
+Also  a nice read for new [Husky Satellite Lab](https://huskysat.org/) members!
```
Comment (nguy8tri) [inline README.md:7]: space!!!

**Review by nguy8tri — CHANGES_REQUESTED** [review]
For some reason, I cannot comment on some of the cctor stuff, so I'll say it here. It looks great, but at the last few sections, its a bit unclear that 1 comes from the heap. I think when you state `Let's assume rank is a pointer to an int allocated on the heap`, it makes it more ambiguous, but this is not needed since `new` "automatically" allocates to the heap, as you'll see in my section (Its not like I just learned this 3 hours ago)! But in general, you can never guarantee that anything points to the heap unless you put an allocating statement (i.e. `malloc()` or `new`) in your constructor for that specific variable, so maybe also make a note about this, because this becomes an issue in the destructor I believe. But great job, your things look good!

File: README.md (hunk 1 above)
Comment (zeddie888) [reply README.md:7]: 😭😭😭😭😭😭😭😭

**Review by nguy8tri — APPROVED** [review]
Perfetto!

**Pull request merged**